src/RamCondition.h

```diff
@@ -147,6 +142,9 @@ class RamNegation : public RamCondition {
     }
 
 protected:
```

Comment:
Any idea why some member variables are protected and some are private? E.g. lhs/rhs are protected in RamConjunction, but lhs/rhs are private in RamConstraint.

Reply:
I don't know why, nor do I see any particular reason for it. This would be a good time to check, though, so I can add that to this PR.

Reply:
Also, RamOperation.h is missing some of the refactoring.

Reply:
There is no reason / it would be great to make member-variables protected in general.